testutils/README.md

```diff
@@ -0,0 +1,3 @@
+# testutils
```

Comment:
aha, might be worth looking at `pytest-pixl` and unifying with that. Having two separate test modules feels overkill

Reply:
Yes, that makes a lot of sense.  At first glance testutils.dicom can be easily moved under pytest-pixl (which didn't exist when I started the branch in my defense!).

Reply:
Ha yep, no need for a defense there!